UI/styles.css

```diff
@@ -0,0 +1,203 @@
+html {
```

Comment:
Please remove the UI files from the pull request on Task 4. They do not belong here